ArduCopter/Parameters.cpp

```diff
@@ -1176,13 +1176,13 @@ const AP_Param::GroupInfo ParametersG2::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("PLDP_THRESH", 1, ParametersG2, pldp_thrust_placed_fraction, 0.9),
 
-    // @Param: PLDP_RNG_MIN
+    // @Param: PLDP_RNG_MAX
     // @DisplayName: Payload Place minimum range finder altitude
```

Comment:
should be maximum in displayname and description

Reply:
fixed